index.html

```diff
@@ -11,21 +11,38 @@
 
 <body>
     <header>
-        Awesome Book Collection
+        Awesome Books <a class="list active" href="">List</a> <a class="add_new" href="">Add new</a> <a class="contact" href="">Contact</a>
     </header>
+    
 
     <main>
         <section id="Book">
-            <div id="book-container"> </div>
-            <div class="addtext">Add a new book</div>
-           
-            <form id= "form">
-                <!-- <label for="title">Title:</label> -->
-                <input placeholder="Title" type="text" name="title" id="title" required>
-                <!-- <label for="author">Author:</label> -->
-                <input placeholder="Author" type="text" name="author" id="author" required>
-                <input type="submit" value="Add" id="add-book">
-            </form>
+            <div class="list-block">
+                <h3 class="allbooks">All awesome books</h3>
+                <div id="book-container"> </div>
+            </div>
+            <div class="add-block">
+                <div class="addtext">Add a new book</div>           
+                <form id= "form">
+                    <!-- <label for="title">Title:</label> -->
+                    <input placeholder="Title" type="text" name="title" id="title" required>
+                    <!-- <label for="author">Author:</label> -->
+                    <input placeholder="Author" type="text" name="author" id="author" required>
+                    <input type="submit" value="Add" id="add-book">
+                </form>
+            </div>
+        </section>
+        <section class="contactsection">
+            <h3>Contact information</h3>
+            <p>Do you have any questions or you just want to say "Hello"?
+                You can reach out to us!
+            </p>
+            <ul>
+                <li>Our email: mail@mail.com</li>
+                <li>Our phone number: 001122334455</li>
+                <li>Our address: Streetname 22, B4503 City, Country</li> 
+            </ul>
+
         </section>
```

Comment:
- [ ] Good job with your project implementation :smile:  But there are some issues you need to address. From the project requirement, you should use a library - [Luxon](https://moment.github.io/luxon/#/install) to display the current date below the navigation bar. Follow this article on how  [get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Please fix it cause this is very important in your developer job you will probably be asked to use external tools :+1:  

#### Project Wireframe
![Screenshot from 2022-08-03 18-34-40](https://user-images.githubusercontent.com/6615394/182685465-1a4f15e3-b3e2-45e2-a134-c7ef7ef7fb9b.png)

#### Your design
![Screenshot from 2022-08-03 18-43-02](https://user-images.githubusercontent.com/6615394/182685459-4532b5ad-4b9e-4890-90a9-ec5c6fa6e3ba.png)